src/examples/radio/RadioExamples.tsx

```diff
@@ -58,6 +58,58 @@ export default function CheckboxExamples () {
           </GoARadioGroup>
         </GoAFormItem>        
       </Sandbox>
+      <h3 id="component-example-expand-collapse-form">Radio item with max width</h3>
```

Comment:
Needs a different id than the above example, it's used when creating the links in the right side nav